apps/dot-voting/app/components/VoteDetails/CastVote.js

```diff
@@ -166,7 +153,7 @@ const CastVote = ({ onVote, toggleVotingMode, vote, voteWeights }) => {
           mode="strong"
           onClick={handleVoteSubmit}
           disabled={
-            voteOptions.reduce((sum, { trueValue = 0 }) => sum + trueValue, 0) === 0
+            voteOptions.reduce((sum, { trueValue = 0 }) => parseInt(sum) + trueValue, 0) === 0
```

Comment:
I think this is probably in the wrong place. We initialize `sum` to `0`, and then we add `trueValue` from each `voteOption`. I think we want to coerce `trueValue` to an int.

I just recently wrote this code, and I'd love ideas on how to make it less awkward. I don't like that we're doing this logic on every render pass; can you think of a more straightforward way to do this?